chall06/sadawi.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+
+import sys, fileinput, re
+
+def get_biggest(shelfs):
+		biggest = 0
+		for i in range(0, len(shelfs) - 1):
+			if int(shelfs[i]) > int(shelfs[biggest]):
+				biggest = i
+		size = shelfs[biggest]
+		del shelfs[biggest]
+		return int(size)
+
+if len(sys.argv) < 2:
```

Comment:
I would suggest having a `if __name__ == "__main__"

and add this protection into it.

https://stackoverflow.com/questions/419163/what-does-if-name-main-do